### ISSUE TITLE
500 Error

### ISSUE THREAD

**Issue by iamkhattar** [SleepyBandit/oxford-dictionary#8]
Hey Sleepy,
Amazing work with the wrapper. I am getting a 500 error with “error parsing json string” when I use the directions from the readme. Could you please look into this?
Thanks x

**Comment by SleepyBandit** [comment]
readme has been updated to reflect changes for API v2. Sorry for the delay hopefully this cleared things up.

**Comment by SleepyBandit** [comment]
Wanted to provide a bit extra info to test as possible cause. The wrapper had pretty generic error handling so any status code other than 404 would result in the 500 error parsing json string code. I've updated the code to handle the different error codes mentioned by the Oxford team. 
https://developer.oxforddictionaries.com/documentation/response-codes

I discovered my own issue was my credentials were still using the FREE account to access the API. The Oxford team mentions in the following documentation that the FREE account cannot access the V2 API!! LAME!!! but that may be your issue.
https://developer.oxforddictionaries.com/version2

If that isn't your problem then the wrapper should return a more useful error code now.
